### PR TITLE
[Changelog] Add Gaprindashvili-2

### PR DESCRIPTION
**Pull request by JPrause**
[skip ci]

### CODE REVIEW

**Comment by JPrause** [comment]
@miq-bot add_label documentation

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,11 +5,15 @@ All notable changes to this project will be documented in this file.
...
+
+### Fixed
+- Fix refresh for stack without parameters [(#410)](https://github.com/ManageIQ/manageiq-providers-amazon/pull/410)
+
 ## Unreleased as of Sprint 80 ending 2018-02-26
 
 ### Fixed
 - Raise Exception if AMI not visible [(#411)](https://github.com/ManageIQ/manageiq-providers-amazon/pull/411)
-- Fix refresh for stack without parameters [(#410)](https://github.com/ManageIQ/manageiq-providers-amazon/pull/410)
 - Return created topic [(#409)](https://github.com/ManageIQ/manageiq-providers-amazon/pull/409)
```
Comment (bronaghs) [inline CHANGELOG.md]: @JPrause - Im curious why this was moved?

Comment (JPrause) [reply CHANGELOG.md]: @bronaghs in this case, the fix in PR410, went from unreleased (not in an upstream official release) to being backported for inclusion in an official upstream release. Only select PRs are backported based on PM review and/or blocker status.

Comment (bronaghs) [reply CHANGELOG.md]: Ah i see, thanks for clarifying.

**Pull request merged**